Source/WebKit/UIProcess/API/mac/WKWebViewMac.mm

```diff
@@ -1470,6 +1470,8 @@ - (BOOL)_ignoresNonWheelEvents
 
 - (void)_setIgnoresNonWheelEvents:(BOOL)ignoresNonWheelEvents
 {
+    if (ignoresNonWheelEvents)
```

Comment:
I'd rather we log unconditionally and add the new value to the log line.

Reply:
Good point, will fix.

Thanks for reviewing!